app/Http/Requests/PositionFormRequest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace App\Http\Requests;
+
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Http/Requests/PositionFormRequest.php</br>**Line**: 26</br>Opening brace should be on a new line